travis.sh

```diff
@@ -0,0 +1,38 @@
+#/bin/bash
+
+set -e
+
+# Travis CI clones the repository at ~/vmware-samples/go-redis-pmem
+# Move the repository into GOPATH first.
+cd ..
+mkdir -p $GOPATH/src/github.com/vmware-samples
+mv go-redis-pmem $GOPATH/src/github.com/vmware-samples
```

Comment:
What if there is already a go-redis-pmem at $GOPATH/src/github.com/vmware-samples?